andela/denethor-ah-frontend
# #164395483 Setup Article Read Page

## What this PR does
- Setup Article Read Page
- Includes Header
- Includes Footer
- Includes Home Banner

## How to test locally
- Clone repo
- Check out branch `feature/164395483/single-article-view-page`
- `npm run dev-server`
- Visit http://localhost:8080/articles/1

### [Pivotal Tracker Story]
(https://www.pivotaltracker.com/story/show/164395483)
